elastic/logs/tasks/package-asset-setup.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "validate-package-template-installation",
+  "tags": ["setup"],
+  "operation": {
+    "operation-type": "validate-package-assets",
+    "packages": {{ p_packages }},
+    "asset-types": ["index-templates"]
```

Comment:
I thought about validating that other asset types are available (e.g. pipelines) but was on the fence about how useful/safe that would be. I'm not 100% sure that every package will install pipelines or ILM policies, but they certainly will install templates. I opted to just implement this for index templates in this PR, but left this param here as an extension point. We can kill it if it's confusing, though.